flake-module.nix

```diff
@@ -402,9 +437,9 @@ in
   };
 
   config.flake = rec {
-    homeModules = readModules cfg.home.modulesDirectory;
-    nixosModules = readModules cfg.nixos.modulesDirectory;
-    darwinModules = readModules cfg.darwin.modulesDirectory;
+    homeModules = injectEarly cfg.home.earlyModuleArgs (readModules cfg.home.modulesDirectory);
+    nixosModules = injectEarly cfg.home.earlyModuleArgs (readModules cfg.nixos.modulesDirectory);
+    darwinModules = injectEarly cfg.home.earlyModuleArgs (readModules cfg.darwin.modulesDirectory);
```

Comment:
I think this is actually wrong, and should be 
```
    homeModules = injectEarly cfg.home.earlyModuleArgs (readModules cfg.home.modulesDirectory);
    nixosModules = injectEarly cfg.nixos.earlyModuleArgs (readModules cfg.nixos.modulesDirectory);
    darwinModules = injectEarly cfg.darwin.earlyModuleArgs (readModules cfg.darwin.modulesDirectory);
```

Reply:
Thank you for the catch, it should be fixed in commit 0444784